back/app/models/permission.rb

```diff
@@ -44,6 +44,7 @@ class Permission < ApplicationRecord
   validates :permission_scope_type, inclusion: { in: SCOPE_TYPES }
 
   before_validation :set_permitted_by_and_global_custom_fields, on: :create
+  before_validation :set_global_custom_fields
```

Comment:
The global custom fields are being set three times now. What was wrong with the original code?

Reply:
Just realised my mistake here, I left something in that shouldn't be there and the second one should be on update

Reply:
Just fixing

Reply:
Removed the repetition and change to on: :update and changed the name to make clearer